client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -472,6 +472,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 	const getBadge = ( themeId: string, isLockedStyleVariation: boolean ) => (
 		<ThemeTierBadge
 			canGoToCheckout={ false }
+			isThemeShowcase
```

Comment:
Nitpick (non-blocking): This is a bit confusing, because the `unified-design-picker.tsx` file is part of the Design Picker, not the Theme Showcase.

What are you trying to achieve when `isThemeShowcase` is true? It seems that the intention is to support two different styles, one for when there is a list of themes and another one for when only one theme is visible.

Maybe you could rename the prop to something else like `isSingleTheme` or `isThemeList`, since that's generic enough for both the Design Picker (`/setup/onboarding/designSetup`) and the Theme Showcase (`/themes`).